aurorastation.dme

```diff
@@ -2367,6 +2367,8 @@
 #include "maps\exodus\code\exodus_holodeck.dm"
 #include "maps\exodus\code\exodus_shuttles.dm"
 #include "maps\exodus\code\exodus_unittest.dm"
+#include "maps\runtime\runtime-1.dmm"
+#include "maps\runtime\runtime-2.dmm"
```

Comment:
Already talked to you about this in discord, but going to comment here too for references' sake.

_Never_ do this, even for testing. The world/map code expects there to be nothing but 1 Z of space at boot, and things like Z-level definitions will get fucked up.

To load a map, add a `FORCE_MAP` directive to your config, or tick one of the dm files in the `_maps` dir (note the underscore).